contributing/branching-and-release-strategy.md

```diff
@@ -0,0 +1,79 @@
+# Branching and release strategy
+
+The Valtimo product aims to deliver features to its user quickly. Contributions from outside the core
+development team are welcomed. To make this possible, the below branching and release strategy is used.
+
+## Branching
+
+### `next-minor`
+
+Both [valtimo-frontend-libraries](https://github.com/valtimo-platform/valtimo-frontend-libraries) and [valtimo-backend-libraries](https://github.com/valtimo-platform/valtimo-backend-libraries/)
+are mainly developed in one single branch: `next-minor`. As its name implies, all commits made into `next-minor` will be
+released publicly when the next minor version of the product is released.
+
+### `next-major`
+
+Breaking changes for the product are developed in the `next-major` branch. Only changes related to the breaking change
+are made here. Unrelated changes are still committed to `next-minor`.
+
+While `next-major` is in development, `next-minor` is frequently merged into `next-major`.
+
+Once the breaking changes in `next-major` are finished and tested, `next-major` is merged into `next-minor`. This
+commit is tagged and released. Work then continues in the `next-minor` branch until a new breaking change needs to be
+made, and a new `next-major` branch is created from `next-minor`.
+
+### Back-end libraries
+
+Because of security reasons, back-end features are developed in feature branches, branched off from `next-minor`. 
+Feature branches are named `feature/*`. Parts of the feature are developed in branches prefixed with `story/`,
+which include a story number. For example: `story/12345-add-endpoint`. These story branches are merged into
```

Comment:
I wouldn't mention the story number here. For external contributions, that wouldn't work out as they do not have story numbers.